saleor/app/types.py

```diff
@@ -6,19 +6,36 @@ class AppType:
 
 
 class AppExtensionView:
+    """All available place where app's iframe can be mounted.
+
+    PRODUCT - app's iframe will be mounted in product section
+    """
+
     PRODUCT = "product"
 
     CHOICES = [(PRODUCT, "product")]
 
 
 class AppExtensionType:
+    """All available type where app's iframe can be mounted.
+
+    OVERVIEW - app's iframe will be mounted on list view.
+    DETAILS - app's iframe will be mounted on detail view
+    """
+
     OVERVIEW = "overview"
     DETAILS = "details"
 
     CHOICES = [(OVERVIEW, "overview"), (DETAILS, "details")]
 
 
 class AppExtensionTarget:
+    """All available places where app's iframe can be mounted.
+
+    MORE_ACTIONS - three dots button
```

Comment:
It doesn't have to be three dots in UI forever:
```suggestion
    MORE_ACTIONS - more actions button
```

Reply:
Should we call these enums `MORE_ACTION_BTN` and `CREATE_BTN`? Is it likely that we will have more options in the future that won't be buttons, but maybe some sections? If so it might be helpful to mention that these refer to buttons.